### PR TITLE
Improve Stack docs

### PR DESCRIPTION
**Pull request by abarth**
Discuss how to change paint order.

### CODE REVIEW

**Comment by abarth** [comment]
@HansMuller

File: packages/flutter/lib/src/widgets/basic.dart [hunk 1]
```diff
@@ -1461,6 +1461,13 @@ class BlockBody extends MultiChildRenderObjectWidget {
 /// placed relative to the stack according to their top, right, bottom, and left
 /// properties.
 ///
+/// The stack paints its children in order. If you want to change the order in
+/// which the children paint, you can rebuild the stack with the children in a
```
Comment (rwozniak) [inline packages/flutter/lib/src/widgets/basic.dart]: typo - `a` at the end of line is not needed

Comment (abarth) [inline packages/flutter/lib/src/widgets/basic.dart]: Thx

File: packages/flutter/lib/src/widgets/basic.dart [hunk 2]
```diff
@@ -1461,6 +1461,13 @@ class BlockBody extends MultiChildRenderObjectWidget {
...
+/// the new order. If you reorder the children in this way, consider given the
+/// children non-null keys. These keys will cause the framework to move the
+/// underlying objects for the hildren to their new locations rather than
```
Comment (rwozniak) [inline packages/flutter/lib/src/widgets/basic.dart]: typo - `hildren` -> `children`

Comment (abarth) [inline packages/flutter/lib/src/widgets/basic.dart]: Fixed.

File: packages/flutter/lib/src/widgets/basic.dart [hunk 3]
```diff
@@ -1461,6 +1461,13 @@ class BlockBody extends MultiChildRenderObjectWidget {
 /// placed relative to the stack according to their top, right, bottom, and left
 /// properties.
 ///
+/// The stack paints its children in order. If you want to change the order in
+/// which the children paint, you can rebuild the stack with the children in
+/// the new order. If you reorder the children in this way, consider given the
```
Comment (rwozniak) [inline packages/flutter/lib/src/widgets/basic.dart]: `given` -> `giving`?

**Review by rwozniak — APPROVED** [review]

**Comment by abarth** [comment]
Thanks!

**Pull request merged**